src/krb5_plugin/common/radius_kdcpreauth.c

```diff
@@ -18,6 +18,9 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#ifdef __FreeBSD__
+#include <sys/param.h>
```

Comment:
Since b928dbe1fcb6cf4b7acbd97d3df9514d5c554953 , `"util/util.h"` included below, includes `<sys/param.h>`

But what in radius_kdcpreauth.c requires it?